astropy/units/quantity.py

```diff
@@ -841,7 +887,7 @@ def to(self, unit, equivalencies=[]):
         --------
         to_value : get the numerical value in a given unit.
         """
-        # We don't use `to_value` below since we always want to make a copy
+        # We don't use `to_value` below since we always want to make a Copy
```

Comment:
Why this change? Just keep as is.

Reply:
My bad have fixed this in the recent pull request
@mhvk can you review it?

Reply:
I still see a change here. Can you undo it?